### ISSUE TITLE
ElastiCache Auto-discovery client for the application

### ISSUE THREAD

**Issue by oksana-c** [department-of-veterans-affairs/va.gov-cms#4819]
## Acceptance Criteria

- [ ] Memcache is not a single point of failure (SPOF)

## Tasks

- [ ] figure out whether it's w/ HA client or shunting
- [ ] ideally use DB cache if Memcache fails

**Comment by ndouglas** [comment]
Relevant core issues: 

https://www.drupal.org/project/drupal/issues/2429321
https://www.drupal.org/project/drupal/issues/2766509 (marked dupe of above)

**Comment by ndouglas** [comment]
>ideally use DB cache if Memcache fails

An issue I encountered and alluded to during the hackathon is that I believe Drupal doesn't clear the DB cache if Memcache exists and is in place.  That DB cache remains in place and drifts out of sync with the filesystem as changes accumulate.  Almost immediately, it ceases to be reliable, especially if any sort of module update takes place.

So I think any fallback from memcache to DB cache needs to be accompanied by a cache clear (I've had better luck with Drush than Drupal Console), and possibly more violent remedies like directly truncating the `cache_*` DB tables, etc.

**TL;DR**: Note to self: test some demanding scenarios here if we retain that task in this issue.

**Comment by timcosgrove** [comment]
Decision was made not to move forward. Closing. https://dsva.slack.com/archives/C02HX4AQZ33/p1661900173643889
